_data-prepper/pipelines/configuration/processors/aggregate.md

```diff
@@ -18,12 +18,139 @@ The following table describes the options you can use to configure the `aggregat
 group_duration | No | String | The amount of time that a group should exist before it is concluded automatically. Supports ISO_8601 notation strings ("PT20.345S", "PT15M", etc.) as well as simple notation for seconds (`"60s"`) and milliseconds (`"1500ms"`). Default value is `180s`.
 
-<!---## Configuration
+## Available aggregate actions
 
-Content will be added to this section.--->
+Use the following aggregate actions to determine how the `aggregate` processes events in each group.
+
+### remove_duplicates 
+
+The `remove_duplicates` action processes the first event for a group immediately and drops any events from the source that duplicate the first. For example, when using `identification_keys: ["sourceIp", "destination_ip"]`:
+
+1. The `remove_duplicates` action processes `{ "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "status": 200 }`, the first event in the source.
+2. Data Prepper drops the `{ "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "bytes": 1000 }` event because the `sourceIp` and `destinationIp` matches the first event in the source.
+3. The `remove_duplicates` action processes the next event `{ "sourceIp": "127.0.0.2", "destinationIp": "192.168.0.1", "bytes": 1000 }`. Because the `sourceIp` is different from the first event of the group, Data Prepper creates a new group based on the event.
+
+### put_all
+
+The `put_all` action combines events belonging to the same group by overwriting existing keys and adding new keys, similar to the Java `Map.putAll`. The action drops all events that make up the combined event.  For example, when using `identification_keys: ["sourceIp", "destination_ip"]`, the `put_all` action processes the following three events:
+
+```
+{ "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "status": 200 }
+{ "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "bytes": 1000 }
+{ "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "http_verb": "GET" }
+```
+
+Then, the action combines the events into one. The pipeline then uses the following combined event:
+
+```
+{ "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "status": 200, "bytes": 1000, "http_verb": "GET" }
+```
+
+### count
+
+The `count` event counts events belong to the same group and generates a new event with values of from the `identification_keys` and the count, which indicates the number of new events. You can customize the processor with the following configuration options:
+
+
+ * `count_key`: Key name to use for storing the count. Default name is `aggr._count`.
+* `start_time_key`: key name to use for storing the start time. Default name is `aggr._start_time`.
+* `output_format`: Format of the aggregated event.
+     * `otel_metrics` - Default output format. Outputs in OTel metrics SUM type with count as value
+    * `raw` - Generates a JSON object with `count_key` field with count as value and `start_time_key` field with aggregation start time as value.
```

Comment:
I'm not following this sentence. Can we simplify/clarify? Are we just missing articles?

Reply:
```suggestion
    * `raw` - Generates a JSON object with the `count_key` field as a count value and the `start_time_key` field with aggregation start time as value.
```